tests/run.py

```diff
@@ -170,9 +170,9 @@ def run_cmd(cmd, expected_to_pass, errstring='Sanity check : ', tmpdir=None, pri
             sys.stderr.write(errstring+ ' -- failed command -- '+cmd+'\n')
             if(not dont_capture_output):
                 sys.stderr.write('STDOUT:\n')
-                sys.stderr.write(stdout_string)
+                sys.stderr.write(stdout_string.decode('utf-8'))
                 sys.stderr.write('STDERR:\n')
-                sys.stderr.write(stderr_string)
+                sys.stderr.write(stderr_string.decode('utf-8'))
```

Comment:
I think there are probably a couple other places where we need this change...`run_cmd` for instance? I think it only gets triggered in error conditions so that's why we're not seeing the issue right now

Reply:
OK, will look for all these instances now... Thanks.